src/inmanta/protocol/methods_v2.py

```diff
@@ -1357,3 +1357,73 @@ def set_password(username: str, password: str) -> None:
     :raises NotFound: Raised when the user does not exist
     :raises BadRequest: Raised when server authentication is not enabled
     """
+
+
+@typedmethod(
+    path="/unmanaged/<unmanaged_resource_id>",
+    validate_sid=False,
+    operation="POST",
+    agent_server=True,
+    arg_options=methods.ENV_OPTS,
+    client_types=[ClientType.agent],
+    api_version=2,
+)
+def unmanaged_resource_create(env: uuid.UUID, unmanaged_resource_id: str, values: Dict[str, str]) -> None:
```

Comment:
The convention is to use the `tid` to reference the environment id in the API endpoint. We should use consistent naming here and also in the other API endpoints.
```suggestion
def unmanaged_resource_create(tid: uuid.UUID, unmanaged_resource_id: str, values: Dict[str, str]) -> None:
```